### ISSUE TITLE
Poloniex error

### ISSUE THREAD

**Issue by presence35** [mobnetic/BitcoinChecker#430]
I can't get a list of trade pairs.  Just says unknown error. Fresh install on fresh Nexus 6P running Pure Nexus 7.1.2

**Comment by MaciejCiemiega** [comment]
Works perfectly fine for me. Can you please check it on DataModuleTester?

**Comment by springfirenator** [comment]
@MaciejCiemiega  The issue is real. The reason it works for you is because only few countries are affected. Poloniex displays captcha when accessing api/site in certain countries. Please resolve this asap, thanks.

**Comment by presence35** [comment]
@springfirenator. You're right! I just tried a USA VPN and it works. Native connection from Ukraine fails.

**Comment by MaciejCiemiega** [comment]
@presence35 It would be useful for me if you can grab the website source and screenshot of the website variant with captcha. I assume that after providing captcha it works? If yes, then how long?

**Comment by presence35** [comment]
No sense posting all that here.  Just go to www.poloniex.com  The very first page is the captcha.

Usually it lasts for maybe 10 minutes or so in Chrome. Then I have to reconfirm the captcha, but I won't have to login again. I didn't check this behaviour in your app. I assume it's the same.

**Comment by MaciejCiemiega** [comment]
@presence35 No, I do not see captcha no matter how many times I'd try. That's why I cannot do anything if I cannot experience it.
Anyway... who puts a captcha on API address? This is ridiculous, API by definition is meant to be used by bots, apps and scripts, not by humans. Have you tried contacting them regarding this issue?

**Comment by presence35** [comment]
You're not getting a captcha cos you're in the US of A.  Use s VPN to be in
Ukraine or Russia or something.

And yeah, valid point about the API access.  I'll send them an email, see
if we can tackle this from 2 angles.

**Comment by MaciejCiemiega** [comment]
Cool, lets see how they will respond to that 👍 
FYI I'm from Poland.

**Comment by presence35** [comment]
Oh, we're neighbours. Nice)

Well this is waht Polo said, so no help there

"We are sorry you are experiencing this problem, unfortunately the captcha
is not something we can control. The best solution would be to use a VPN to
connect to the API."

**Comment by zhtet036** [comment]
zhtett036
